src/web/app/components/question-responses/grouped-responses/grouped-responses.component.scss

```diff
@@ -14,3 +14,7 @@
   margin-left: auto;
   height: max-content;
 }
+
+.bottom-border {
```

Comment:
Is there a typo? I don't this class being used?

Reply:
It's being used at the top of the html file
https://github.com/TEAMMATES/teammates/blob/eaceb1116a0434d7aa091c5d22c635afe09d122c/src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html#L1

Reply:
But that is `border-bottom` though, not `bottom-border`?

Reply:
ah yes typo sorry, thanks for catching that